### PR TITLE
Database - Stored Proc, Backup/Recovery Scripts

### PR DESCRIPTION
**Pull request by JLaveta**
Changes in this request:

Removed procedures_db.sql
Removed general_queries.sql
Fixed a couple of issues with the database populate script.
Fixed and tested both the backup and recover scripts. Provided usage instructions.
Added more stored procedures, specifically to eventClassProcs.sql
Added drops to stored procedure scripts.

Requesting review from both CI members since I can't review my own pull.

### CODE REVIEW

**Comment by JLaveta** [comment]
Not sure why we have those conflicts, but those are the two files that can be deleted, so feel free to toss them.

**Comment by mviencek** [comment]
@JLaveta do you want us to delete those two files from dev?
 models/database/general_queries.sql
models/database/procedures_db.sql

**Comment by JLaveta** [comment]
Yes, please. Thanks!

**Comment by mviencek** [comment]
deleted.  thank you for cleaning up

File: models/database/stored_procs/organizationClassProcs.sql [hunk 1]
```diff
@@ -9,20 +15,19 @@ WHERE organizationid = organizationIdp;
 
 END//
 
-#deletes an organization and all related details based off organizationId
+#deletes an organization and a user from the organizationId
 
-CREATE PROCEDURE `deleteOrginization` (IN orginizationIdpd int)
+CREATE PROCEDURE `deleteOrganization` (IN organizationIdd int(11), UserId int(11))
```
Comment (mviencek) [inline models/database/stored_procs/organizationClassProcs.sql]: why are you passing in userid if you arent using it

File: models/database/stored_procs/eventsClassProcs.sql [hunk 2]
```diff
@@ -338,6 +268,204 @@ END//
...
+-- public static function removeAddress($eventId, $address)
+    -- {
+        -- //remove an address from the event in the database
+
+        -- //categories, types, and addresses likely need their own
+        -- //methods for changes since they can have multiple entries
+    -- }
+
+
+CREATE PROCEDURE removeAddress(IN currentEventId INT, currentAddressId INT, currentAddressLine1 VARCHAR(45), currentAddressLine2 VARCHAR(45), currentCity VARCHAR(45), currentState VARCHAR(45), currentPostalCode VARCHAR(45))
```
Comment (mviencek) [inline models/database/stored_procs/eventsClassProcs.sql]: a lot of your sprocs are passed in extra parameters that you dont seem to be using

Comment (JLaveta) [reply models/database/stored_procs/eventsClassProcs.sql]: Thanks for pointing that out. I'll follow up with the people who wrote this and see if we can get it corrected today.

**Comment by JLaveta** [comment]
@mviencek @cjshaw This should be ready to pull in now.

**Review by mviencek — APPROVED** [review]
great. thank you.  for your next submit.  can you guys combine everything into two seperate scripts.  One that drops if exists and creates for sprocs and the  same thing for the database,tables, and sample data?

**Comment by mviencek** [comment]
just the sproc one.  i missed that you are already doing that with the db

**Comment by JLaveta** [comment]
We're not currently populating the database with the same script used to create it. But yes, we'll plan to combine everything this coming week so we have one create/populate script and one stored procedures script.

**Review by cjshaw — APPROVED** [review]

**Pull request merged**